lab1/print_style.css

```diff
@@ -0,0 +1,83 @@
+body {
+  padding: 50px 100px;
+  background-color: white;
+  font-family: "Comic Sans MS", arial;
+  font-size: 100%;
+  text-align: justify;
+  text-justify: inter-word;
+}
+nav {
+  display:none;
+}
+
+a {
+  text-align: center;
```

Comment:
Should all links really be centered? Maybe we have links in text, might not be the best solution.
Better idea is maybe to have a `center` class which does this.